packages/client/src/components/Auth/Signin.jsx

```diff
@@ -16,17 +23,18 @@ export const Signin = () => {
     setError('');
     try {
       await signIn(email, password);
-      // TODO: Display a modal
-      navigate('/');
+      setIsModalOpen(true);
+      setTimeout(() => {
+        navigate('/');
+      }, 1500);
     } catch (err) {
-      // error
       setError(err.message);
+      setIsModalOpen(true);
+      setTimeout(() => {
+        navigate('/signin');
```

Comment:
Is this the /auth? Or should /auth be this? 
AFAIK /signin is currently a 404 page, right?

Reply:
Changed the paths to match